test/src/unit-cppapi-config.cc

```diff
@@ -113,3 +113,16 @@ TEST_CASE(
   const std::string result3 = config[key];
   CHECK(result3 == value3);
 }
+
+TEST_CASE("C++ API: Config Equality", "[cppapi], [cppapi-config]") {
+  // Check for equality
+  tiledb::Config config1;
+  config1["foo"] = "bar";
+  tiledb::Config config2;
+  config2["foo"] = "bar";
+  CHECK((config1 == config2) == true);
```

Comment:
While correct and entirely subjective, we typically would just do `CHECK(config1 == config2)` here. Likewise for L127: `CHECK(config1 != config2)`. Let's do that to stay consistent with the rest of the codebase.